dnf/cli/commands/deplist.py

```diff
@@ -29,7 +29,8 @@ class DeplistCommand(RepoQueryCommand):
     """
 
     aliases = ('deplist',)
-    summary = _("List package's dependencies and what packages provide them")
+    summary = _("(deprecated) List package's dependencies and what packages provide them "
+                "(use repoquery --deplist instead)")
```

Comment:
A quick grep shows we don't seem to have any deprecated notices in help strings so far. I'm in favor of this, just to bikeshed on the actual format (which we should use consistently from now on) I'd probably do it like this:
> [deprecated, use repoquery --deplist] List package's dependencies and what packages provide them

Potentially put the `[]` notice at the end.

Reply:
Sounds good. I also changed commit message, because I had forgotten about it before.